doc/api/changes_dnf5.2.rst

```diff
@@ -0,0 +1,64 @@
+###############################################
+ Modifications to the public API in dnf 5.2.0.0
+###############################################
+
+This page lists the differences in the public API of `DNF5 5.2 <https://github.com/rpm-software-management/dnf5/releases/tag/5.2.0.0>`_ compared to the previous major version, DNF5 5.1.
+
+
+Wrapping struct attributes
+==========================
+
+Public attributes in struct objects have been wrapped in getter and setter methods.
+
+Before: ``bool ignore_case;`` Now: ``void set_ignore_case(bool); bool get_ignore_case() const;``
+
+* ``libdnf5::GoalJobSettings``
```

Comment:
It isn't immediately obvious what all this encompasses. For example, when `group_with_name` stops working I search the changelog for mention of it, not of GoalJobSetting.

Detailing all the changes would help when searching.

Reply:
I've added a file and linked it to the docs with diff between dnf5-5.1.17 and dnf5-5.2.0.0 API changes. I guess that would be better for the text search.